tests/test_compat.py

```diff
@@ -51,7 +51,7 @@ def test_text_alternate_encoding(self):
         self.assertTrue(isinstance(result, bytes))
         self.assertEqual(result, b'La Pe\xc3\xb1a')
 
-class Test_cgi_FieldStorage_Py3_tests(object):
```

Comment:
We use py.test for testing, this does not need to be sub-classed from unittest.TestCase.